action.yml

```diff
@@ -76,11 +53,43 @@ runs:
         echo "PLAYGROUND_PASSWORD=${{ inputs.PLAYGROUND_PASSWORD }}" >> $GITHUB_ENV
         echo "PLAYGROUND_ID=${{ inputs.PLAYGROUND_ID }}" >> $GITHUB_ENV
         echo "DOCASSEMBLE_DEVELOPER_API_KEY=${{ inputs.DOCASSEMBLE_DEVELOPER_API_KEY }}" >> $GITHUB_ENV
+        echo "ALKILN_VERSION=${{ inputs.ALKILN_VERSION }}" >> $GITHUB_ENV
       shell: bash
     - name: "ALKiln: confirm info"
       run: echo -e "\nRepo is $REPO_URL\nBranch ref is $BRANCH_PATH\nManually added languages are $EXTRA_LANGUAGES\nAll other data is secret\n"
       shell: bash
 
+    # This doesn't use an already established package-lock.json and that
+    # comes with some types of test breaking risks. Same problem as before.
+    - name: Overwrite or create package.json
+      run: |
+        cat << EOF > "package.json"
+        {
+          "scripts": {
+            "test": "npm run langs_setup && npm run cucumber -- $@",
+            "cucumber_base": "./node_modules/.bin/cucumber-js --require ./node_modules/@suffolklitlab/alkiln/lib/index.js",
+            "cucumber": "run_cucumber(){ npm run cucumber_base -- \"$@\" docassemble/*/data/sources/*.feature; }; run_cucumber",
+            "langs": "npm run langs_setup && ./node_modules/.bin/cucumber-js --require ./node_modules/.bin/alkiln-cucumber docassemble/*/data/sources/*_lang*.feature",
+            "langs_setup": "node_modules/.bin/alkiln-langs 'docassemble/*/data/sources/*.feature'",
+            "setup": "node_modules/.bin/alkiln-setup",
+            "takedown": "node_modules/.bin/alkiln-takedown"
+          },
+          "dependencies": {
+            "colors": "1.4.0",
+            "@cucumber/cucumber": "^7.3.2",
+            "@suffolklitlab/alkiln": "$ALKILN_VERSION"
+          }
+        }
+      shell: bash
+
+    # Install node
+    - name: "ALKiln: Install node packages"
+      uses: actions/setup-node@v1
+      with:
+        node-version: "17"
+    - run: npm install
+      shell: bash
```

Comment:
illegitimate nitpick: it is out-of-scope of this PR, but I believe https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#defaultsrun will remove the need of repeating the `shell: bash`.

Reply:
#528